tests/unittest/test_breakpad_resource.py

```diff
@@ -204,6 +204,12 @@ def test_existing_uuid(self, client):
         # throttleable is not 0
         ({'Throttleable': '1'}, (0, 'has_comments', 100)),
         ({'Throttleable': 'foo'}, (0, 'has_comments', 100)),
+
+        # throttler rejects the code and it stays rejected regardless of throttleable value
+        (
+            {'Throttleable': '0', 'HangID': '555', 'ProcessType': 'browser'},
```

Comment:
The hangid rule is easier to simulate and it produces a REJECT, so I used that here. Any REJECT rule would do fine.